playbooks/common/openshift-cluster/upgrades/pre/verify_upgrade_targets.yml

```diff
@@ -14,24 +14,26 @@
 - when: not openshift.common.is_containerized | bool
   block:
   - name: Check latest available OpenShift RPM version
-    command: >
-      {{ repoquery_cmd }} --qf '%{version}' "{{ openshift.common.service_type }}"
-    failed_when: false
-    changed_when: false
-    register: avail_openshift_version
+    repoquery:
```

Comment:
It should but it is already imported by other play. The verify stuff should go into a role most likely. Then, it will have lib_utils as its dependency.